### PR TITLE
ci: setup Cypress tests

### PR DESCRIPTION
**Pull request by arturovt**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,17 +5,25 @@
...
+    "build:integration:ghpages": "ng build --configuration ghpages --no-progress",
+    "serve:integration:static": "serve dist -s -l 4200 --cors",
     "build-data": "tsx scripts/build-data.ts",
     "test": "ng test --watch=false",
     "test:watch": "ng test",
     "test:ci": "ng test --watch=false --code-coverage --no-progress --browsers=ChromeHeadlessCustom",
     "link": "cd dist && npm link",
     "lint": "ng lint",
     "lint:fix": "ng lint --fix",
-    "ghpages": "ng build --configuration production --no-progress"
```
Comment (scttcper) [inline package.json:18]: this doesn't actually deploy to github pages anymore, it deploys the demo to vercel i just never changed the name

Comment (arturovt) [reply package.json:18]: should we change it to smth like `build:vercel`?

Comment (scttcper) [reply package.json:18]: i guess 🤷

Comment (arturovt) [reply package.json:18]: updated

**Pull request merged**